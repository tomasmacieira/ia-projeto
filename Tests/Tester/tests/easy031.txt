ROW	0	1	2	6	1	2	0	2	2	4
COLUMN	3	0	5	1	1	3	1	3	2	1
9
HINT	2	2	T
HINT	7	9	W
HINT	4	2	M
HINT	5	8	C
HINT	5	2	B
HINT	9	5	L
HINT	9	6	M
HINT	2	0	B
HINT	3	8	M
