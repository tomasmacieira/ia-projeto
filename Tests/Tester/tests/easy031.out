..........
t.........
B.T.......
..m.lr.lMr
..M.......
..B.....C.
..........
.....c.c.W
..lr......
c....LMr..
